src/main/java/seedu/resireg/logic/parser/AllocateCommandParser.java

```diff
@@ -30,14 +30,9 @@ public AllocateCommand parse(String args) throws ParseException {
                     String.format(MESSAGE_INVALID_COMMAND_FORMAT, AllocateCommand.HELP.getFullMessage()));
         }
 
-        try {
-            Index studentIndex = ParserUtil.parseIndex(argMultimap.getValue(PREFIX_STUDENT_INDEX).get());
-            Index roomIndex = ParserUtil.parseIndex(argMultimap.getValue(PREFIX_ROOM_INDEX).get());
-            return new AllocateCommand(studentIndex, roomIndex);
-        } catch (ParseException pe) {
-            throw new ParseException(
-                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, AllocateCommand.HELP.getFullMessage()), pe);
-        }
+        Index studentIndex = ParserUtil.parseIndex(argMultimap.getValue(PREFIX_STUDENT_INDEX).get());
+        Index roomIndex = ParserUtil.parseIndex(argMultimap.getValue(PREFIX_ROOM_INDEX).get());
+        return new AllocateCommand(studentIndex, roomIndex);
```

Comment:
I think it shouldn't be the case that the portion on throwing ParseException should be removed since we need the parser to throw an exception if parsing fails. For the invocation of ParseException, I think just testing using `assertParseFailure` and `assertParseSuccess` would be sufficient.

Reply:
Fixed.